style.css

```diff
@@ -261,22 +265,149 @@ body.lock {
     font-size: 20px;
     right: 25px;
 }
-.arrow_left {
+.arrow-left {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(135deg);
 }
-.arrow_right {
+.arrow-right {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(-45deg);
 }
 .content {
-    padding: 70px 0 0;
+    margin: 60px 0 0;
+}
+.carousel {
+    position: relative;
+    width: 100vw;
+    height: 150vw;
+    overflow: hidden;
+}
+.carousel__text {
+    position: absolute;
+    top: 50%;
+    left: 8%;
+    text-align: left;
+    display: active;
+    z-index: 20;
+}
+.carousel__text1 {
+    color: var(--element-color);
+    font-size: 9vw;
+    letter-spacing: 3px;
+}
+.carousel__text2 {
+    color: var(--additional-color);
+    font-size: 8vw;
+    letter-spacing: 1.5px;
+}
+.carousel__box {
+    position: absolute;
+    transition: 0.5s;
```

Comment:
There is no transition here. What does it transition?

Reply:
There is transition - it is slide transition in a carousel.

Reply:
What does it transition? Last time I checked you have to specify what the transition applies to.

Reply:
This transition applies to carousel__box's that contain images (slides), that change their position left.

Reply:
Got it, changed